### PR TITLE
Adds a diagram for the perf setup script

### PR DESCRIPTION
**Pull request by danehans**
This PR adds a diagram to `tools/README.md` explaining the deployment details of the `setup_perf_cluster.sh` script.

Fixes: https://github.com/istio/istio/issues/3495

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
 # Istio Load Testing User Guide
+### Introduction
 This guide provides step-by-step instructions for using the `setup_perf_cluster.sh` load testing script.
-The script deploys a GKE cluster, an Istio service mesh, a GCE VM and runs [Fortio](https://github.com/istio/fortio/)
-on the VM and within the mesh. Fortio is used to perform load testing, graph results and as a backend echo server.
+The script deploys a GKE cluster, an Istio service mesh and a GCE VM. The script then runs [Fortio](https://github.com/istio/fortio/)
+on the VM, 2 pods within the cluster (non-Istio) and 2 pods within the Istio mesh. The following diagram provides
+additional details of the deployment:
+
+![Deployment Diagram](https://user-images.githubusercontent.com/1878195/36280330-c37e420e-124e-11e8-9ca0-3ad42443f0a0.png)
```
Comment (ldemailly) [inline tools/README.md]: can you add the svg source and link it instead ?

**Comment by danehans** [comment]
@ldemailly I added the svg. I amended the commit instead of adding a new commit... sorry about that.

**Comment by ldemailly** [comment]
ty!

**Pull request merged**